tests/Browser/AdminListDailyMenuTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Tests\Browser;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Facebook\WebDriver\Chrome\ChromeOptions;
+use \App\DailyMenu;
+use \App\Food;
+use \App\Category;
+
+
+class AdminListDailyMenuTest extends DuskTestCase
+{
+    use DatabaseTransactions;
+    use WithoutMiddleware;
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test URL admin view list daily menu.
+     *
+     * @return void
+     */
+    public function testURL()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(1)
+            ->visit('/dashboard')
+            ->clickLink('Management')
+            ->clickLink('Daily Menu Management')
+            ->assertPathIs('/daily-menus');
+        });
+    }
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test show content.
+     *
+     * @return void
+     */
+    public function testContent()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(1)
+                ->visit('/daily-menus')
+                ->assertSee("LIST DAILY MENUS");
+        });
+    }
+
+    /**
+     * @group dailymenu
+     *
+     * A Dusk test count object in list=10.
+     *
+     * @return void
+     */
+    public function testPaginationWith10Rows()
```

Comment:
You want to test list has one page right?. If correct, you should be change function name to `testHasOnePage`. Limit per page may be changing, don't keep 10 forever.